Updates to managing listings by action (#397)
magento/merchdocs: src/sales-channels/asc/publish-listings-manually.md

---
title: Publish an Amazon Listing
---

## Publish a single or multiple Amazon listing that has been ended

1. View the listing(s) on the _Ended_ tab on the [Product Listings]({% link sales-channels/asc/managing-product-listings.md %}) page (_Inactive_, _Active_, or _Ineligible_ tab).

> 1. In the left-side column, click to check each of the listings you wish to republish.

Reviewer: ```suggestion
1. In the left column, click to check each of the listings you want to republish.
```

1. Under _Actions_, click **Publish Product to Amazon**.

1. Click **OK** on the confirmation message.

## Publish a single Amazon listing that has been ended

1. View the listing(s) on the _Ended_ tab on the [Product Listings]({% link sales-channels/asc/managing-product-listings.md %}) page (_Inactive_, _Active_, or _Ineligible_ tab).

1. In the _Action_ column, click **Select** > **Publish On Amazon**.

1. Click **OK** on the confirmation message.

    A confirmation message will display that the selected listing is being processed to publish to Amazon.

>     Listing information is published to Amazon based on your cron settings. Listing information will send to Amazon at the next data sync. Until Amazon replies with the listing confirmation, the manually published listing(s) remains on the _Ready to List_ tab with a `List in Progress` status. Once the listing confirmation is received from Amazon, the listing(s) moves to the _Active_ tab with an `Active` status.

Reviewer: ```suggestion
    Listing information is published to Amazon based on your cron settings. Listing information is sent to Amazon at the next data sync. Until Amazon replies with the listing confirmation, the manually published listing(s) remains on the _Ready to List_ tab with a `List in Progress` status. When the listing confirmation is received from Amazon, the listing(s) moves to the _Active_ tab with an `Active` status.
```
